pkgs/development/libraries/libfprint/default.nix

```diff
@@ -1,14 +1,19 @@
-{ stdenv, fetchurl, pkgconfig, meson, ninja, libusb, pixman, glib, nss, gtk3
+{ thinkpad ? false, stdenv, fetchurl, pkgconfig, meson, ninja, libusb, pixman, glib, nss, gtk3
 , coreutils, gtk-doc, docbook_xsl, docbook_xml_dtd_43 }:
 
 stdenv.mkDerivation rec {
   name = "libfprint-${version}";
   version = "0.99.0";
 
-  src = fetchurl {
-    url = "https://gitlab.freedesktop.org/libfprint/libfprint/uploads/82ba3cef5bdf72997df711eacdb13c0f/libfprint-${version}.tar.xz";
-    sha256 = "16r4nl40y0jri57jiqmdz4s87byblx22lbhyvqpljd6mqm5rg187";
-  };
+  src = fetchurl (
```

Comment:
You should probably use fetchFromGitHub here

Reply:
Really? https://github.com/NixOS/nixpkgs/pull/27327#issuecomment-314797154 says I shouldn't.